### PR TITLE
create cart and order -> master

### PR DESCRIPTION
**Pull request by datnguyen0126**

### CODE REVIEW

File: src/components/CartLine.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    sendUpdateEvent() {
+      this.$emit("update", {
+        food_id: this.item.food.id,
+        quantity: this.qvalue,
+      });
+    },
+  },
+};
+</script>
```
Comment (ntdo2506) [inline src/components/CartLine.vue]: new line

File: src/components/Cart.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <Navbar />
+    <div class="container">
+      <table id="cart" class="table table-hover table-condensed">
+        <thead>
+          <tr>
+            <th style="width:48%">Product</th>
+            <th style="width:10%">Price</th>
+            <th style="width:8%">Quantity</th>
+            <th style="width:22%" class="text-center">Subtotal</th>
+            <th style="width:12%"></th>
```
Comment (ntdo2506) [inline src/components/Cart.vue:12]: style is not in html tag

File: src/components/OrderSummary.vue [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<script>
+    export default {
+        computed: {
+            orderId() {
+                return this.$route.params.id;
+            }
+        }
+    }
+</script>
```
Comment (ntdo2506) [inline src/components/OrderSummary.vue]: new line

File: src/components/Cart.vue [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        <tfoot>
+          <tr>
+            <td>
+              <router-link to="/" class="btn btn-warning">
+                <i class="fa fa-angle-left"></i>Back to shop
+              </router-link>
+            </td>
+            <td colspan="2" class="hidden-xs"></td>
+            <td class="hidden-xs text-center">
+              <strong>Total ${{ items.reduce((total, item) => total + (item.quantity * item.price), 0) }}</strong>
```
Comment (Taythaytu0905) [inline src/components/Cart.vue:37]: I think shoud you data variable, Not use Js in HTML

File: src/components/Checkout.vue [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+              <li class="list-group-item d-flex justify-content-between bg-light">
+                <div class="text-success">
+                  <h6 class="my-0">Discount</h6>
+                </div>
+                <span class="text-success">-{{ (item.food.discount * item.total_price).toFixed(2) }}</span>
+              </li>
+            </div>
+            <li class="list-group-item d-flex justify-content-between">
+              <span>Total (USD)</span>
+              <strong>{{items.reduce((total, item) => total + (item.quantity * item.price * (1 - item.food.discount)), 0).toFixed(2)}}</strong>
```
Comment (Taythaytu0905) [inline src/components/Checkout.vue:53]: I think shoud you data variable, Not use Js in HTML

File: src/components/FoodtList.vue [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        'food_id': food_id,
+        'quantity': quantity
+      }
+      axios
+        .post(url, data, {
+          withCredentials: true,
+          headers: authHeader()
+        })
+        .then((resp) => {
+          console.log(resp.data);
```
Comment (ntdo2506) [inline src/components/FoodtList.vue]: remove console.log

**Pull request closed without merging**